packages/cli/src/commands/apps/delete.ts

```diff
@@ -12,6 +12,11 @@ export default class AppDeleteCommand extends APICommand<typeof AppDeleteCommand
 		description: 'App profile UUID or number in the list',
 	}]
 
+	static examples = [
+		{ description: 'select app to delete from list', command: 'smartthings apps:delete' },
+		{ description: 'delete a specific app by id', command: 'smartthings <app-id>' },
```

Comment:
Should this be `smartthings apps:delete <app-id>`?

Reply:
Yes it should but this wasn't supposed to be in the PR. I've fixed this PR to not include it at all and I'll update in PR #422.

Reply:
Updated in PR #422 now.